### PR TITLE
V2: CLI Batchnorm

### PR DESCRIPTION
**Pull request by KnathanM**
We never exposed the batch norm option for models to the CLI. 

Note that the misspelling of `exta_mpnn_args` (should be `extra_mpnn_args`) will be fixed in #675

### CODE REVIEW

**Comment by davidegraff** [comment]
you probably want this to be `True` by default because it improves training so much.

**Comment by KnathanM** [comment]
Good catch, I agree. It made the implementation a bit more complex as it needs a double negative (`not --no-batch-norm`), but `--no-batch-norm` seems to be self explanatory.

**Review by hwpang — APPROVED** [review]
LGTM
